packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1360,7 +1446,7 @@ describe('KeyringController', () => {
         );
         const recovered = recoverTypedSignature({
           data: typedMsgParams,
-          signature,
+          signature: signature.toString(),
```

Comment:
These `toString()` calls can be removed; this should be a string already with the latest keyring controller version